pcap4j-sample/pom.xml

```diff
@@ -17,6 +17,10 @@
   <dependencies>
     <dependency>
       <groupId>org.pcap4j</groupId>
+      <artifactId>pcap4j-packetfactory-static</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.pcap4j</groupId>
```

Comment:
This dependency is not really needed, I think. A dependency should be added only when it's needed to build or unit tests.